---
title: 'Zero-inflated Conway-Maxwell mixed model: An application to Microbiome data'
summary: A Comparison between ZICMP and ZINB models.
tags:
  - Zero Inflated Models
date: '2024-01-08T15:00:00Z'

# Optional external URL for project (replaces project detail page).
external_link: ''

image:
  caption: Oral Presentation Certificate
  focal_point: Smart

links:
  - icon: linkedin
    icon_pack: fab
    name: Follow
    url: https://www.linkedin.com/in/abhiram-d-b-69094221b/
url_code: ''
url_pdf: ''
url_slides: ''
url_video: ''

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: example
---

### INTRODUCTION
Microbiome count data is typically a representation of the abundance or count of each microbial taxon within a given sample. In the realm of microbiome research, when a researcher aims to construct a regression model using count-based microbiome data, several challenges emerge. It predominantly includes zero inflation which entails the presence of zeros in a dataset. In scenarios involving count data with an excess of zeros, the utilization of single distribution models isn’t feasible. Instead, employing zero-inflated regression models becomes necessary, as they can effectively manage both the data generation process and the presence of excess zero.
### OBJECTIVE
To compare Zero Inflated Negative Binomial Model (ZINB) and Zero Inflated Conway-
Maxwell Poisson Model (ZICMP) in handling cross-sectional microbiome data.
### METHODOLOGY
The performance of both models was evaluated using a secondary dataset from a study at NIMHANS involving 60 subjects categorized into drug-naïve or risperidone-treated Schizophrenia patients and healthy controls. The study utilized a 16s ribosomal RNA (16s rRNA) gene sequence-based method to explore gut microbiome differences as a potential non-invasive biomarker for Schizophrenia. In addressing overdispersion and excess zeroes in the count data, we applied the ZINB and ZICMP models. Model selection was based on various criteria, including Akaike Information Criteria (AIC), Vuong’s Test, and Rootogram plots for comparison.
### RESULTS
Both models demonstrate comparable performance, as indicated by their AIC values of 190.5 for both the ZINB and ZICMP models. Furthermore, the Root Mean Squared Error (RMSE) values are very close, with 288.56 for ZINB and 288.67 for ZICMP. These results suggest similar predictive accuracy for both models.
### CONCLUSION
Both models provide comparable fits and predictive performance for the given dataset. 
#### Keywords: Microbiome, ZINB, ZICMP, Vuong’s Test, Rootogram, RMSE
